CHANGES.rst

```diff
@@ -26,6 +26,12 @@ combine_1d
 - Fixed a bug to properly exclude input spectra that have only 1
   wavelength bin. [#7053]
 
+cube_build
```

Comment:
Now that 1.8.0 has been released, this entry needs to be moved up to the new 1.8.1 section.